server/src/main/java/com/hedera/block/server/persistence/storage/write/BlockAsDirWriterBuilder.java

```diff
@@ -68,16 +68,17 @@ public static BlockAsDirWriterBuilder newBuilder(
      * and directories.
      *
      * <p>By default, the block writer will use the permissions defined in {@link
-     * FileUtils#defaultPerms}. This method is primarily used for testing purposes. Default values
-     * should be sufficient for production use.
+     * com.hedera.block.common.utils.FileUtilities#DEFAULT_DIR_PERMISSIONS}. This method is
+     * primarily used for testing purposes. Default values should be sufficient for production use.
```

Comment:
Just remove this paragraph, it is describing internal details of a different library.
Also, defaults are never acceptable for production in this case; defaults are for testing and development; configuration is for production.

Reply:
:white_check_mark: Done.